public/app/config/search/museum/resultFields.js

```diff
@@ -47,10 +51,18 @@
                     labelText: 'Classification',
                     displayValue: function (value) {
                         return $('<ul class="list-unstyled"></ul>')
-                            .append(_(JSON.parse(value))
+                            .append(_(parse(value))
                                 .map(function (valueItem) {
-                                    return $('<li></li>').html(_(valueItem).drop().join(' &rArr; '));
+                                    return $('<li></li>').html(_(valueItem)
+                                        .drop()
+                                        .map(function (hierarchyNodeItem, depth) {
+                                            return depth === 0 ?
+                                                hierarchyNodeItem :
+                                                _.repeat('&nbsp;', 2 * depth) + '&rArr; ' + hierarchyNodeItem;
```

Comment:
Still not sure if I like this vs nested `ul`s, but as it's only single items at each level, it's probably fine.